src/request/components/request-detail-panel-logging.jsx

```diff
@@ -26,16 +27,16 @@ var getMessages = (function() {
  */
 function getRowClass(message) {
```

Comment:
The CSS is still applying certain properties (e.g. foreground color), so it's still needed.  If it's possible and reasonable (and that's just showing my ignorance of CSS) to put that all in CSS, than that's fine with me, too.